src/components/reusable/progressBar/progressBar.ts

```diff
@@ -0,0 +1,273 @@
+import { html, LitElement } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+import '@kyndryl-design-system/shidoka-foundation/components/icon';
+import '../loaders';
+import '../tooltip';
+import checkmarkIcon from '@carbon/icons/es/checkmark--filled/16';
+import errorIcon from '@carbon/icons/es/error--filled/16';
+import informationIcon from '@carbon/icons/es/information/16';
```

Comment:
I was going to say we should use Shidoka icons here, but I don't think there's an error icon yet.